Documentation/doc/resources/1.8.13/cgal_stylesheet.css

```
@@ -339,3 +339,9 @@ div.fragment {
     padding: 4px;
     margin: 1em 4px 1em 4px;
 }
+
+/* Make summary smaller to avoid wrapping of classes and concepts */
+div.summary
+{
+  width: 20%;
```

Why `20%`? With `auto` instead, it will only take the size it needs, and nothing more.

```suggestion
  width: auto;
```